include/mrdox/Metadata/Type.hpp

```diff
@@ -34,24 +36,16 @@ namespace mrdox {
     @li unsigned long, unsigned long long
 */
 struct TypeInfo
-    : Reference
 {
-    TypeInfo() = default;
+    SymbolID id;
+    std::string Name;
 
     explicit
     TypeInfo(
-        Reference const& R) noexcept
-        : Reference(R)
-    {
-    }
-
-    // Convenience constructor for when there is no symbol ID or info type
-    // (normally used for built-in types in tests).
-    explicit
-    TypeInfo(
-        std::string_view Name)
-        : Reference(
-            SymbolID::zero, Name, InfoKind::Default)
+        SymbolID ID = SymbolID::zero,
+        std::string_view name = "") noexcept
```

Comment:
the smallest nitpick imaginable: consider this instead:
```
std::string_view name = {}
```
What do you think?

Reply:
Yeah you're right that's better